gsp_matplotlib/visual/markers.py

```diff
@@ -3,7 +3,7 @@
 # License: BSD 3 clause
 
 import numpy as np
-from gsp import glm
+from .. import glm
```

Comment:
Can't we use explicit name (not fan of .., it's confusing)

Reply:
we can, i did that to make it easier to move file around. 

i will do the change

Reply:
change done